Bottom: c2b37320467865b77cad2d17e9096c6728ffe03f
Top:    24715b4db27963bdcdb321576847c4da74da5c02
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 22:36:41 +0100

Refresh of parse_y-y-don-t-bother-setting

---

diff --git a/src/misc.c b/src/misc.c
index 0b756bd..e2644f8 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -994,8 +994,6 @@ ParseGroupString (char *s, LayerGroupType *LayerGroup, int LayerN)
     s_set = false;              /* provide a default setting for old formats */
   int groupnum[MAX_LAYER + 2];
 
-  printf ("ParseGroupString (\"%s\", %p, %i)\n", s, LayerGroup, LayerN);
-
   /* clear struct */
   memset (LayerGroup, 0, sizeof (LayerGroupType));
 
@@ -1046,8 +1044,6 @@ ParseGroupString (char *s, LayerGroupType *LayerGroup, int LayerN)
           LayerGroup->Entries[group][member++] = layer;
           while (*++s && isdigit ((int) *s));
 
-          printf ("Group %i contains Layer %i\n", group, layer);
-
           /* ignore white spaces and check for separator */
           while (*s && isspace ((int) *s))
             s++;
